### DISCUSSION TITLE
Windows软件折腾之路 | Goalonez

### DISCUSSION THREAD

**Discussion by Goalonez** [Goalonez/goalonez.github.io#159] [Announcements]
https://blog.goalonez.site/blog/Windows%E8%BD%AF%E4%BB%B6%E6%8A%98%E8%85%BE%E4%B9%8B%E8%B7%AF.html 

 Goalonez Blog

**Comment by luxuxl** [comment]
压缩推荐个 Bandzip, 启动器我忘了我用的啥 (好久没回家用 Win 了), 话说博主不折腾改键吗
